src/app/main/component/user/components/profile/profile-dashboard/profile-dashboard.component.ts

```diff
@@ -157,6 +157,13 @@ export class ProfileDashboardComponent implements OnInit, OnDestroy {
       });
   }
 
+  removeUnFavouriteEvent(id: number): void {
+    const indexOfUnFavouriteEvent = this.favouriteEvents.findIndex((event) => event.id === id);
+    if (indexOfUnFavouriteEvent !== -1) {
+      this.favouriteEvents.splice(indexOfUnFavouriteEvent, 1);
+    }
+  }
```

Comment:
I would use filter method here
 removeUnFavouriteEvent(id: number): void {
    this.favouriteEvents = this.favouriteEvents.filter((event) => event.id !== id);
  }

Reply:
Done it